poollet/bucketpoollet/cmd/bucketpoollet/app/app.go

```diff
@@ -100,7 +100,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.LeaderElectionKubeconfig, "leader-election-kubeconfig", "", "Path pointing to a kubeconfig to use for leader election.")
 
 	fs.StringToStringVar(&o.BucketDownwardAPILabels, "bucket-downward-api-label", o.BucketDownwardAPILabels, "Downward-API labels to set on the IRI bucket.")
-	fs.StringToStringVar(&o.BucketDownwardAPIAnnotations, "bucket-downward-api-annotations", o.BucketDownwardAPIAnnotations, "Downward-API annotations to set on the IRI bucket.")
+	fs.StringToStringVar(&o.BucketDownwardAPIAnnotations, "bucket-downward-api-annotation", o.BucketDownwardAPIAnnotations, "Downward-API annotations to set on the IRI bucket.")
```

Comment:
This is a potential breaking change when upgrading `ironcore`.

/cc @damyan @lukasfrank